### ISSUE TITLE
Issue defining ContentItem Action Filters on new Webhook creation

### ISSUE THREAD

**Issue by kswan-hw** [kontent-ai/management-sdk-js#133]
### Brief bug description

While attempting to create a new webhook with content item action filters, the MAPI returns with validation errors because the actions block on the content_item trigger doesn't match the expected format.

### Additional context

In the SDK, the following typescript interfaces are defined:

```
interface IAddWebhookData {
	name: string;
	secret: string;
	url: string;
	last_modified?: string;
	enabled?: boolean;
	delivery_triggers: {
		slot: WebhookDeliveryTriggerSlots;
		events: WebhookDeliveryTriggersEvents;
		asset?: WebhookContracts.IWebhookAssetContract;
		content_type?: WebhookContracts.IWebhookContentTypeContract;
		taxonomy?: WebhookContracts.IWebhookTaxonomyContract;
		language?: WebhookContracts.IWebhookLanguageContract;
		content_item?: WebhookContracts.IWebhookContentItemContract;
	};
}
```

```
interface IWebhookContentItemContract {
	enabled: boolean;
	actions?: WebhookContentItemActions[];
	filters?: IContentItemFilters;
}
```

```
type WebhookContentItemActions = 'published' | 'unpublished' | 'created' | 'changed' | 'metadata_changed' | 'deleted' | 'workflow_step_changed';
```

In this case, the "actions" is defined as an array of strings, but in the MAPI documentation, it appears that it should be an array of an object that contains an "action" and a "transition_to" attribute.

https://kontent.ai/learn/docs/apis/openapi/management-api-v2/#tag/Webhooks

**Comment by Nickm615** [comment]
Working on it now

**Comment by Enngage** [comment]
Released in `5.8.1`
